docs/guide/adding_noise.md

```diff
@@ -0,0 +1,239 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.2
+kernelspec:
+  name: python3
+  display_name: Python 3 (ipykernel)
+  language: python
+---
+
+# Adding Noise
+
+## Motivation
+
+Noise is an essential component of QEC workflows.
+The whole idea of QEC only makes sense if qubits are not perfect.
+Decoders try to make logical qubits robust against errors,
+but in order for them to do a good job we have to inform them which errors they should expect.
```

Comment:
```suggestion
The goal of Quantum Error Correction is to correct errors, which are caused by different noise channels in the system. In order for a decoder to more accurately approximate errors in the system based on a set of error syndromes, it's important to pass the characteristics of these noise channels.
```

Reply:
May you consider my proposal of your edited text.
```suggestion
The goal of Quantum Error Correction is to correct errors, which are caused by different noise mechanisms.
A noise channel is a way to describe a probabilistic change of a quantum state.
In order for a decoder to more accurately correct errors,
it's important to pass a model of the noise, described using noise channels.
```

```